NewFile.txt

```diff
@@ -0,0 +1,7 @@
+This is a new file.
+
+With some new content.
+
+Seriously...
+
+Very important content.
```

Comment:
Reply.

---
In reply to: [70331932](https://github.com/kivancmuslu/GitHubTest/pull/1#discussion_r70331932) [](ancestors = 70331932)